src/PDO/Peachpie.Library.PDO/PDOStatement.cs

```diff
@@ -443,6 +443,11 @@ public virtual PhpArray fetchAll(PDO_FETCH fetch_style = default, PhpValue fetch
                 }
             }
 
+            if (fetch_style.HasFlag(PDO_FETCH.FETCH_CLASS) && !fetch_argument.IsEmpty)
```

Comment:
Please don't use `Enum.HasFlag()`, it's significantly slow.

Following is recommended:
```C#
(fetch_style & PDO_FETCH.FETCH_CLASS) != 0
```

Reply:
Noted.